### PR TITLE
move go-libp2p-mplex here

### PR DESCRIPTION
**Pull request by marten-seemann**
Part of #1187.

### CODE REVIEW

**Review by Stebalien — APPROVED** [review]

File: p2p/transport/websocket/websocket_test.go [hunk 1]
```diff
@@ -15,6 +15,8 @@ import (
 	"testing"
 	"time"
 
+	mplex "github.com/libp2p/go-libp2p/p2p/muxer/mplex"
```
Comment (Stebalien) [inline p2p/transport/websocket/websocket_test.go:18]: do we need the rename?

Comment (marten-seemann) [reply p2p/transport/websocket/websocket_test.go:18]: The package is named `peerstream_multiplex`, so I think we do.

Comment (Stebalien) [reply p2p/transport/websocket/websocket_test.go:18]: So, can we, like, fix that. Now?

Comment (Stebalien) [reply p2p/transport/websocket/websocket_test.go]: Probably.

**Pull request merged**